File: python/01-iniciante/problem1041.py
# Inputs

x, y = input().split()
x = float(x)
y = float(y)

# Outputs

if (x > 0.) and (y > 0.):
    print("Q1")
elif (x < 0.) and (y > 0.):
    print("Q2")
elif (x < 0.) and (y < 0.):
    print("Q3")
elif (x > 0.) and (y < 0.):
    print("Q4")
elif (x != 0) and (y == 0):
    print("Eixo X")
elif (x == 0) and (y != 0):
    print("Eixo Y")
else:
    print("Origem")

File: go/01-iniciante/problem1041.go
package main

import "fmt"

func main() {
    var x, y float64

    fmt.Scanf("%f %f", &x, &y)

    if (x > 0.0) && (y > 0.0) {
        fmt.Println("Q1")
    } else if (x < 0.0) && (y > 0.0) {
        fmt.Println("Q2")
    } else if (x < 0.0) && (y < 0.0) {
        fmt.Println("Q3")
    } else if (x > 0.0) && (y < 0.0) {
        fmt.Println("Q4")
    } else if (x == 0.0) && (y == 0.0) {
        fmt.Println("Origem")
    } else if (x != 0.0) && (y == 0.0) {
        fmt.Println("Eixo X")
    } else {
        fmt.Println("Eixo Y")
    }
}
